### PR TITLE
vmui: add compact JSON display

### PR DESCRIPTION
**Pull request by Loori-R**
### Describe Your Changes
If a JSON element has only one field, it will be displayed on a single line.
 #6559

| Old Display | New Display |
|-------------|-------------|
| image | image |
 
### Checklist

The following checks are **mandatory**:

- [ ] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Review by tenmozes — APPROVED** [review]

File: app/vmui/packages/vmui/src/components/Views/JsonView/JsonView.tsx [hunk 1]
```diff
@@ -12,7 +12,17 @@ export interface JsonViewProps {
 const JsonView: FC<JsonViewProps> = ({ data }) => {
   const copyToClipboard = useCopyToClipboard();
 
-  const formattedJson = useMemo(() => JSON.stringify(data, null, 2), [data]);
+  const formattedJson = useMemo(() => {
+    const space = "  ";
+    const values = data.map(item => {
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/components/Views/JsonView/JsonView.tsx:17]: nit, data can be a null or empty array.

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: app/vmui/packages/vmui/src/components/Views/JsonView/JsonView.tsx (hunk 1 above)
Comment (Loori-R) [reply app/vmui/packages/vmui/src/components/Views/JsonView/JsonView.tsx:17]: `data` cannot be `null`.
In the case of an empty array, no errors will occur.

**Review by hagen1778 — APPROVED** [review]
Thank you!

**Pull request merged**